### DISCUSSION TITLE
RE8 third person view problem.

### DISCUSSION THREAD

**Discussion by Robs40** [praydog/REFramework#565] [General Questions]
Hi, I bought the DLC containing SHADOWS OF ROSE but since there is only the possibility of playing in the third person, the character has his arms behind his body when I am in VR.
Can you fix this problem or should we wait for CAPCOM to add the first person?
![jjj](https://user-images.githubusercontent.com/117215602/199316339-9b0b1c25-5e04-426e-b92d-8cacfc167a4c.jpg)

**Comment by praydog** [comment]
Play with a gamepad.

**Reply by marcell123455** [reply]
> Play with a gamepad.

Aiming won´t work, so that´s not a solution :/

**Reply by Robs40** [reply]
> Jouez avec une manette de jeu.

The fun of playing in VR is being able to use our hands and aim ourselves.
If you can't do anything, I would wait for the first person view to be added by an update from Capcom because many players are already complaining.
